client/desktop/window-handlers/external-links/editor/index.js

```diff
@@ -109,7 +109,7 @@ async function handleJetpackEnableSSO( mainWindow, info ) {
 
 	try {
 		// Allow sufficient time for Gutenberg's "placeholder" UI to render.
-		// Otherwise, exiting the placeholder UI will make Calypso's master- and
+		// Otherwise, exiting the placeholder UI will make Calypso's primary- and
```

Comment:
I think this meant "masterbar" and "sidebar", and wasn't referring to the primary branch (which is still currently called `master`). Let's rephrase to `...masterbar and sidebars disappear`.

Reply:
Ok, now it makes sense :) 
Fixed.